frontend/src/metabase/css/dashboard.css

```diff
@@ -350,7 +350,6 @@
 
 .ScalarValue {
   font-weight: 900;
-  font-size: 1.8rem;
```

Comment:
Deleting these since they are no longer in use. The query builder has its own CSS rule for ScalarValue font size, so it is not affected by any of this.